File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        for(int i=0;i<nums.size();i++){
            for(int c=i+1;c<nums.size();c++){
                if(nums[i]+nums[c]==target){
                    vector<int> arr;
                    arr.push_back(i);
                    arr.push_back(c);
                    return arr;
                } 
            }
        }
        return {};
    }
};

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] dizi= new int[2];
        int n=0;
        for(int i=0;i<nums.length-1;i++){
            n=nums[i];
            for(int j=i+1;j<nums.length;j++){
                if(target-n==nums[j]){  
                    dizi[0]=i;
                    dizi[1]=j;
                    return dizi;
                }
            }
        }
        return dizi;
    }
}